### ISSUE TITLE
Transifex Admin

### ISSUE THREAD

**Issue by cbeams** [bisq-network/roles#20]
See https://www.transifex.com/bisq/public/

Keybase: `#transifex`

Team: @bisq-network/transifex-admins 
Primary: @MwithM    
Secondary: @ripcurlx

**Comment by stevenkain** [comment]
Present for taking over Transifex operations

**Comment by ManfredKarrer** [comment]
Thanks @stevenkain for your offer! At the moment @riclas is the main maintainer. Best to cross check with him if he is interested to become official operator.

**Comment by cbeams** [comment]
Just following up here, that per [conversation in Slack](https://bisq.slack.com/archives/C6YNLRYS2/p1504703703000189), @stevenkain is planning to write up a _specification_ for this role, possibly in collaboration with @riclas if he is also interested. See #21 for an example of a role specification.

**Comment by cbeams** [comment]
I've updated the assignees and description to reflect that @ripcurlx is now the primary role owner here. @ripcurlx, please report monthly, etc as appropriate, thanks.

**Comment by r4el** [comment]
Hello,
Just joined Bisq on Transifex. For reference only, I have published technical papers in English, and performed document translations in Spanish and French in the past. Furthermore, I am interested in supporting Bisq's Transifex platform as admin if you'd like.

**Comment by ManfredKarrer** [comment]
Hi @r4el, 
Thanks for reaching out! That would be great if you could help us there! I assume you are already experienced with Transifex. Can you PM me on Slack or the Forum, then I can bring you in contact with the current Admin...

**Comment by ripcurlx** [comment]
## 2018.05 report
I took over this role as I want to streamline the workflow how we get required translations for new features and to initiate a review of a all existing translations. I applied for a free open source license for Transifex and will have a call today with @r4el on the requirements for this role and possible hand it over at the same time.

/cc https://github.com/bisq-network/compensation/issues/71

**Comment by ManfredKarrer** [comment]
@ripcurlx 
Thanks for taking over that role!
We have the free open source license, but somehow Transifex does not show that properly.
Maybe best it to get in touch with Ricardo who did it so far. I sent an email to connect you both.

**Comment by ripcurlx** [comment]
So far everything seems to be fine with the Transifex open source license. I applied for it within the app settings and nobody complained about it, that we are not fulfilling their requirements.
<img width="1283" alt="bildschirmfoto 2018-06-01 um 15 47 34" src="https://user-images.githubusercontent.com/170962/40844029-2188662e-65b3-11e8-9a1f-b41bbba7907c.png">
Yes, even if Ricardo wants to continue being an admin I think it is no problem to have two people involved in localization efforts.

**Comment by cbeams** [comment]
@ripcurlx, can we get @ManfredKarrer out of the picture completely for this role, even as secondary? Mainly, I mean such that he no longer has keys to Transifex and doesn't have to worry about it anymore. This could be done in conjunction with handing things over to @r4el if that works out, and you could remain as secondary then.

**Comment by ripcurlx** [comment]
## 2018.6 report

Besides continuing onboarding of @r4el, who is currently working on a proposal for translators and reviewer metrics, we synced the translation file between the core repository and Transifex to have only on Transifex the truth. Translation files have been updated in the core repository as well for the new release 0.7.1.
We saw nice activity in June in translations and reviews. Unfortunately we can't drill deeper on the details with our open-source license.
<img width="1020" alt="bildschirmfoto 2018-06-29 um 17 27 39" src="https://user-images.githubusercontent.com/170962/42100975-0387fa3e-7bc2-11e8-8727-cfe0ca6d6241.png">

/cc https://github.com/bisq-network/compensation/issues/84

**Comment by ripcurlx** [comment]
## 2018.07
July was great month in terms of contribution for our transifex project.
<img width="1012" alt="bildschirmfoto 2018-07-30 um 12 55 55" src="https://user-images.githubusercontent.com/170962/43393660-30497838-93f8-11e8-89f3-0bbac2ca5bf4.png">
Besides new translators especially @initCCG pushed the translations for Thai, Vietnamese, Persian and Russian.
<img width="994" alt="bildschirmfoto 2018-07-30 um 12 57 32" src="https://user-images.githubusercontent.com/170962/43393675-3deb966a-93f8-11e8-9101-1a899966cecc.png">
/cc https://github.com/bisq-network/compensation/issues/98

**Comment by ripcurlx** [comment]
## 2018.08
Moderation of the #transifex channel on Slack. We saw less activity in August compared to July, but that is mainly because of @initCCG is nearly done with translating and reviewing the new languages.
<img width="1012" alt="bildschirmfoto 2018-08-30 um 18 41 48" src="https://user-images.githubusercontent.com/170962/44866076-7a297d00-ac84-11e8-9e7a-3fee6fe0f82f.png">
@ripcurlx, @HarryMacfinned and @BravoIndia had a call discussing the problem when a translation/review is up for compensation and how it should be valued. Based on the call [a proposal](https://github.com/bisq-network/proposals/issues/37) was created which should, if it finds consensus, be taken for all upcoming translation/review compensation requests.

**Comment by ripcurlx** [comment]
## 2018.09 report
Moderation of the #transifex channel on Slack. We saw less activity in September compared to August. Translations updates/reviews were done in Thai, Vietnamese, Russian and Spanish.

![bildschirmfoto 2018-09-27 um 11 33 29](https://user-images.githubusercontent.com/170962/46137186-681e0880-c249-11e8-8c7c-5fdf7d9a1a25.png)

We'll see three new languages added in v0.9.0 (Thai, Persian, Vietnamese)

/cc https://github.com/bisq-network/compensation/issues/126

**Comment by ripcurlx** [comment]
## 2018.10 report
Moderation of the #transifex channel on Slack. We saw less activity again in October compared to September. Translations updates/reviews were done in Greek, Persian, Vietnamese, Russian and Portuguese.

![Transifex Report](https://user-images.githubusercontent.com/170962/47710751-d5033480-dc33-11e8-9f0d-ad8b5e1a69cb.png)

We'll see two new languages added in v0.9.0 (Thai, Vietnamese). Persian will be delayed because of not existing consensus between translator and reviewer atm.

/cc https://github.com/bisq-network/compensation/issues/148

**Comment by ripcurlx** [comment]
## 2018.11 report
Moderation of the #transifex channel on Slack. We saw more activity in November compared to October as we switched the source of the translation source file to the new release branch file before merging it to master. This time my work on the German translations will also be part of this compensation request.

<img width="1017" alt="Transifex Report" src="https://user-images.githubusercontent.com/170962/49284882-2c482f00-f496-11e8-8974-9c57f677015d.png">

We'll see two new languages added in v0.9.0 (Thai, Vietnamese). Persian will be delayed because of not existing consensus between translator and reviewer atm.

/cc https://github.com/bisq-network/compensation/issues/177

**Comment by ripcurlx** [comment]
## 2018.12 report
Moderation of the #transifex channel on Slack. We saw more than twice the activity in December compared to November as lots of new strings where translated for our latest releases. This time my work on the German and other translations (translation fixes) will also be part of this compensation request.

<img width="999" alt="bildschirmfoto 2018-12-30 um 12 35 27" src="https://user-images.githubusercontent.com/170962/50546875-6b050c80-0c2f-11e9-8909-172279c5ae1b.png">

/cc https://github.com/bisq-network/compensation/issues/195

**Comment by ripcurlx** [comment]
## 2019.01 report
Moderation of the #transifex channel on Slack. 
I added @arunasurya as Team Manager to be able to work on the Glossary and communicate with other translation managers, translators and reviewers.
We saw less activity in January compared to December as we didn't add lots of new strings or languages to the client. My work on the German translation is also part of this compensation request.

<img width="1013" alt="bildschirmfoto 2019-01-30 um 12 10 25" src="https://user-images.githubusercontent.com/170962/51977629-35e71680-2488-11e9-9848-c063534852ef.png">

/cc https://github.com/bisq-network/compensation/issues/206

**Comment by ripcurlx** [comment]
## 2019.02 report
Moderation of the #transifex channel on Slack. 
We saw less activity in February compared to January as we didn't add again lots of new strings or languages to the client. My work on the German translation is also part of this compensation request.
Next month I'll hand over the Transifex Admin role to @arunasurya. Thanks Aruna for taking over this role!

![bildschirmfoto 2019-02-27 um 12 25 19](https://user-images.githubusercontent.com/170962/53487082-d7af5280-3a8a-11e9-979e-3e5d3ad0d237.png)

/cc https://github.com/bisq-network/compensation/issues/224

**Comment by ManfredKarrer** [comment]
@ripcurlx I think you are handing over to @arunasurya right? Let me know once that is done then I would change the roles. Let me know who should be primary and secondary.

**Comment by ripcurlx** [comment]
> @ripcurlx I think you are handing over to @arunasurya right? Let me know once that is done then I would change the roles. Let me know who should be primary and secondary.

Actually we did the hand over yesterday. Feel free to set @arunasurya as primary and me as secondary role owner already.

**Comment by arunasurya** [comment]
# 2019.03 Report

There was an increase in activity in March. German and Portuguese translations are complete, with Russian and Spanish translations nearing completion.

New initiatives (WIP): standardized glossary across all languages.

![Transifex report pic March 2019](https://user-images.githubusercontent.com/43150241/55290737-6f73c980-53c6-11e9-95f1-43197a5850e5.png)

**Comment by arunasurya** [comment]
# 2019.04 Report

We saw less activity in April compared to March.

![Transifex report April 2019](https://user-images.githubusercontent.com/43150241/56963648-f6909a80-6b48-11e9-8822-3bc04e090248.png)

As of today, the translations in the following languages have been completed or almost completed:

![Language completion](https://user-images.githubusercontent.com/43150241/56963814-6010a900-6b49-11e9-98f4-827444228b87.png)

The current release is available in eleven languages: English, German, Russian, Spanish, Portuguese (Brazil and Portugal), Chinese (Mandarin), French, Persian, Thai, Vietnamese and Greek. Greek will be removed from the new release.

New documentation was created to help the translators and reviewers: 
https://github.com/arunasurya/bisqtransifex

**Comment by arunasurya** [comment]
# 2019.05 Report

There was almost twice as much activity in the bisq-desktop project in May as in April. The translations are complete in the following languages: French, German, Portuguese, Russian and Spanish. Vietnamese is almost complete.

![Tx May 2019 Report_1](https://user-images.githubusercontent.com/43150241/58810997-4a4a4400-861f-11e9-8fd0-d969c55d7613.png)

![Tx_May 2019_Report_2](https://user-images.githubusercontent.com/43150241/58811001-4fa78e80-861f-11e9-857b-9d83a9586a43.png)

We started a new project, bisq-website. At this moment, only homepage is available, and it has been translated into French, German and Spanish.

![TX_May2019_Report_3](https://user-images.githubusercontent.com/43150241/58811310-dc524c80-861f-11e9-865c-5ca855da0f03.png)

**Comment by y3v63n** [comment]
## 2019.06 Report

**Bisq-desktop:** The activity hasn't changed much since May, mainly due to the lack of new content in the application. French, German, Portuguese, Russian and Spanish remain fully translated. We are looking for new translators for Vietnamese and Thai to complete the translations. Persian is currently on hold due to the right-to-left language issue.

<img width="689" alt="2019-07-01 (2)" src="https://user-images.githubusercontent.com/39760876/60419563-3ed24480-9bee-11e9-9623-c8ba1ea359ce.png">

**Bisq-website:** The activity increased six-fold compared to May. Portuguese is 100% translated, while Spanish, French and German are nearing completion.

<img width="684" alt="2019-07-01 (4)" src="https://user-images.githubusercontent.com/39760876/60419839-eb142b00-9bee-11e9-9b2f-e8625abc031b.png">

We have seen an increased activity among new translators, with a total of six people joining Arabic, Spanish, Persian, Hebrew, Portuguese language teams in June.

**Comment by bisq-github-admin-3** [comment]
I've just added @y3v63n as primary role owner as @arunasurya stepped temporary back from this role. Thanks @y3v63n for taking over the role! 👍

**Comment by y3v63n** [comment]
**Bisq-desktop:** We had a huge spike in translation activity in July due to inflow of several new contributors. We now have Chinese, French, German, Portuguese, Russian and Spanish fully translated. We also launched a [bounty](https://github.com/bisq-network/proposals/issues/104) to transtale the Bisq software into Japanese, and the translation is now 80% complete.

![Opera Снимок_2019-08-01_091540_www transifex com](https://user-images.githubusercontent.com/39760876/62270371-9575ad00-b43e-11e9-9dae-3d603cf7cffd.png)

**Bisq-website:** The activity keeps increasing. Chinese, Portuguese Spanish and German are now fully translated, French and Persian are nearing completion.

![Opera Снимок_2019-08-01_091714_www transifex com](https://user-images.githubusercontent.com/39760876/62270647-277db580-b43f-11e9-9acd-b2bfd0843405.png)

**New Translators:** Five new people joined the Chinese, Japanese and German teams. We are still actively looking for Vietnamese and Thai translators.

**Comment by y3v63n** [comment]
## 2019.08 Report ##

**Bisq-desktop:** The translation activity in August dropped due to lack of new content. The Bisq software is now completely translated into Chinese, French, German, **Japanese**, Portuguese, Russian and Spanish.

![Opera Снимок_2019-09-02_145814_www transifex com](https://user-images.githubusercontent.com/39760876/64113380-d55be700-cd92-11e9-8a2b-9acd22c1e2de.png)

**Bisq-website:** The translation activity is declining due to the lack of new content. At the same time, translations that neared completion are now ready, and we now have Chinese, French, German, Portuguese, Persian and Spanish fully translated. Japanese and Russian are over 50% ready.

![Opera Снимок_2019-09-02_145849_www transifex com](https://user-images.githubusercontent.com/39760876/64113731-b6aa2000-cd93-11e9-9351-c3bb4bf927db.png)

**New Translators**: Five new people joined the Portuguese (Brazil), Spanish, Vietnamese and French teams.

**Comment by y3v63n** [comment]
## 2019.09 Report

**Bisq-desktop:** The translation activity in September increased compared to August. The source file was updated at the end of the month to reflect the latest changes in the software, so none of the translations are now fully completed.

![Opera Снимок_2019-10-02_063352_www transifex com](https://user-images.githubusercontent.com/39760876/66018431-5cf65c00-e4df-11e9-99db-a36c0efb336c.png)

**Bisq-website:** The translation activity almost tripled in September. Translations to most of the core languages are now almost completed.

![Opera Снимок_2019-10-02_063311_www transifex com](https://user-images.githubusercontent.com/39760876/66018558-e3ab3900-e4df-11e9-89e3-a2648227fcaf.png)

**Bisq-subtitles:** A new project was added to Bisq Transifex organization after several translation team members expressed willingness to translate some of the video subtitles to their langauges.

![Opera Снимок_2019-10-02_063439_www transifex com](https://user-images.githubusercontent.com/39760876/66018615-279e3e00-e4e0-11e9-9282-0a777c869258.png)

**New Translators:** Four new people joined the Portuguese (Brazil), French and German teams.

**Comment by MwithM** [comment]
There's been relevant activity, so I'm going to make a little report as Spanish Coordinator:
Thanks to Oscarml for notifying that transifex wasn't updated. After the source was updated, he translated all 3700 new words and I reviewed them, in a few days.
Some strings (768 words) related to very low volume altcoins are not being translated on purpose because it feels that they're going to be delisted (if they aren't already). 
We missed Juan Tellez, I'd like to hear from him soon.

**Comment by arunasurya** [comment]
# 2019.11 Report

I have taken over the role of the Transifex admin from @y3v63n in November. 

There was a decline in activity across all projects. There were several language requests in the projects. The languages that support the Bisq app have been accepted while others are pending (to be discussed during the November translator call).

## All projects

![Transifex November Report_allprojects](https://user-images.githubusercontent.com/43150241/69591474-c9c33980-0fea-11ea-8fcb-94b7f8f9f2ad.png)

## bisq-desktop
![Transifex report Nov 19_desktop](https://user-images.githubusercontent.com/43150241/69591491-d0ea4780-0fea-11ea-9aa2-a2853b582dd9.png)

## bisq-website
![Transifex report Nov 2019 website](https://user-images.githubusercontent.com/43150241/69591500-d8a9ec00-0fea-11ea-90cb-aa587652d564.png)

## Bisq Subtitles
![Transifex report Nov 19 subtitles](https://user-images.githubusercontent.com/43150241/69591511-de073680-0fea-11ea-977b-f44646e13f60.png)

**Comment by arunasurya** [comment]
# 2020.12 Report (cycle 9)

Two languages were added to the list of core languages: Italian and traditional Chinese (Taiwan). 

Total BSQ spent on the project (cycle 8): 9,350.99

Breakdown:
Chinese (China and Taiwan): 4,281.32
Transifex admin (primary): 2900
Spanish: 593.95
Russian: 555.14
Technical support (adding Chinese, etc): 500
French: 361.58
Website work: 100
Portuguese: 59

There was a slight increase in activity in the desktop & website projects, and an increase in the subtitles.

![Jan 2020 desktop](https://user-images.githubusercontent.com/43150241/72757992-6d832d00-3bc9-11ea-8b87-ba5504f89173.png)

![Jan 2020 website](https://user-images.githubusercontent.com/43150241/72758000-7411a480-3bc9-11ea-9db3-3c7e1267466e.png)

![Jan 2020 subtitles](https://user-images.githubusercontent.com/43150241/72758006-78d65880-3bc9-11ea-8958-ede398836228.png)

**Comment by MwithM** [comment]
As the new Transifex Admin, I've created a [wiki page](https://bisq.wiki/Translating_Bisq) with instructions and objectives for the next 2 or 3 cycles.

This is the planned budget for Cycle 11:

|   	| Budget  	| 
|--:	|---	|
|  BRA | 250  	|
|  Update | 200  	|
|  Jp 	| 150  	|
|  Admin 	| 100  	|
|   	| 750  	|

While Brazilian translation should be almost 100% completed, Japanese budget is only for about 2000 words. Update languages which are almost completely translated (ES, FR, DE, PT, CH). Languages have been prioritized considering current trade volume they're providing to Bisq. You can follow the tasks at the [kanban board](https://grow.bisq.cc/b/pS7ZjPr2A5oEAvD5D/translations) and Transifex.

**Comment by MwithM** [comment]
After compensation requests have been accepted, expenses are like this:

![imagen](https://user-images.githubusercontent.com/50149324/77821976-d2d62b80-70ee-11ea-83e9-6d3dc40c6042.png)

Surplus is $303. Assuming I can use the budget not spent on previous cycles, the budget limit for cycle 12 is $1053. It will be used to complete translations for Japanese and Italian, and keep up to date the rest, as planned on [kanban board](https://grow.bisq.cc/b/pS7ZjPr2A5oEAvD5D/translations).

![imagen](https://user-images.githubusercontent.com/50149324/77822614-b76e1f00-70f4-11ea-8fa7-f9647912428d.png)

**Comment by MwithM** [comment]
Before latest post, I assumed that all italian bisq-desktop translations were already compensated. They aren't, and $1108 worth of strings tranlsated are left to be compensated. $803.67 to Riccardo, $224.14 to GreGab and 224 to 8uDD4T.
There's a huge backlog for Italian translations. I don't want to keep other languages stopped because of strings translated before new [budget ](https://github.com/orgs/bisq-network/teams/dao/discussions/9)system was running.
Italian budget is going to be lightly expanded for this cycle, but there's going to be work left to be paid for next ones. Compensation requests for tasks that can't be compensated in this cycle because of low budget should be requested at this cycle anyway, to better estimate next cycle budget requirements.

This is the new budget for Cycle 12:

![imagen](https://user-images.githubusercontent.com/50149324/78029580-23ed5600-7361-11ea-82c5-e9fd54049ce6.png)

Please reffer to me at keybase #transifex channel so I can update [Kanban board](https://grow.bisq.cc/b/pS7ZjPr2A5oEAvD5D/translations) as soon as possible in order to know how much of the planned work is going to be done.

**Comment by MwithM** [comment]
## Cycle 12 report

Italian has a huge payout for compensation requests backlog and looks like chinese has too. All core languages have been up to date, except Portuguese. 
Japanese has not been translated in this period.

I'm moving the kanban board from bisq.cc to a cryptopad.fr public board.

**Comment by MwithM** [comment]
On cycle 13, Japanese should be finally translated, Italian will be available for bisq-desktop and Czech too (thanks to a voluntary contributor, as Czech is not a core language) and the rest of languages except Russian will be kept up to date. 
This is the [budget](https://cryptpad.fr/sheet/#/2/sheet/view/yyTkVJCDmOlqo+JxVNDGIgimnf2+mS2ufbxtX4A+IpY/) for cycle 13:

![image](https://user-images.githubusercontent.com/50149324/80689370-84d98c80-8acd-11ea-8a5f-d1e0aa16d1a7.png)

The translation board has been moved to [cryptpad](https://cryptpad.fr/kanban/#/2/kanban/edit/NEP7CObyAx73QrL+TzIK6Zxl/). Contributors can edit the board without registering.

**Comment by MwithM** [comment]
Only Spanish and German has been taken up to date. Japanese translation is stalled, but it might be good to save this to have funds to translate, when ready, the new website [frontpage](https://deploy-preview-367--bisq-website.netlify.app/getting-started/) to multiple languages.
Italian and Czech desktop translations have not been uploaded to the app. It's postponed to next Cycle, as ripcurlx has now something much better to do :-)

**Comment by MwithM** [comment]
On cycle 14 I plan to keep up to date the core languages, and wait for the new website.
Italian and Czech translations could be uploaded on this cycle. The [translation budget spreadsheet](https://docs.google.com/spreadsheets/d/1DdzjrQtD9PAt2wbkUp4z3AwAJhv1JI9S/edit#gid=893684999) has been moved to google drive, as cryptpad spreadsheets are not very reliable yet.

![imagen](https://user-images.githubusercontent.com/50149324/83781889-6ac43880-a68f-11ea-91d8-e29641665de7.png)

**Comment by MwithM** [comment]
## For cycle 14

Activity has been almost null.

**Comment by MwithM** [comment]
As there's been no activity lately, the tasks for cycle 15 are the same as for cycle 14.

![imagen](https://user-images.githubusercontent.com/50149324/86743026-dc015d80-c038-11ea-80c6-91658e9234c3.png)

I'm studying to start translating the[ getting started](https://bisq.network/getting-started/) page.

**Comment by MwithM** [comment]
## For Cycle 15.

Deutsche and Spanish bisq-desktop has been taken up to date and I started trying the app to translate the wiki, for Spanish.
https://bisq.wiki/Payment_methods/es

**Comment by MwithM** [comment]
### Budget for Cycle 16:

![image](https://user-images.githubusercontent.com/50149324/89622166-e52d5680-d892-11ea-8897-d16886a11b11.png)

#### Tasks

- Keep core languages bisq-desktop translations up to date, look for translators.
- Implement Italian and Czech languages with @ripcurlx 
- Define budget per language and what to translate for getting-started pages at wiki.

##### _Budget spreadsheet can be found [here](https://docs.google.com/spreadsheets/d/1DdzjrQtD9PAt2wbkUp4z3AwAJhv1JI9S/edit#gid=893684999)._

**Comment by MwithM** [comment]
### Cycle 16 report

None of the tasks on the previous post have been completed.

**Comment by MwithM** [comment]
## Budget for cycle 17

Due to very little activity, only 16USD was spent in last cycle. This is the budget for cycle 17:

![imagen](https://user-images.githubusercontent.com/50149324/92378129-967b1280-f105-11ea-9bdb-2c2abb516ff2.png)

#### Budget link: https://docs.google.com/spreadsheets/d/1DdzjrQtD9PAt2wbkUp4z3AwAJhv1JI9S/edit#gid=979011045

**Comment by MwithM** [comment]
## Cycle 17 report

- At least French and Japanese are back to being worked on.
- Website is also ready for translation in this cycle (I will update the wiki to announce it).
- I looked at getting-started pages wiki to stablish a budget, but still have many doubts about it.

**Comment by MwithM** [comment]
### Budget for Cycle 18

There's been active translators for Japanese and Chinese (which has been up to date). With new features and changes, new strings are being uploaded to translate and allocated budget for bisq-desktop increases. This is bisq-desktop estimated expenses left.

![imagen](https://user-images.githubusercontent.com/50149324/95329272-fbfa2480-08a6-11eb-93a3-b86a04dd1491.png)

Translation for bisq-website will only take place for Chinese and Spanish. Most of untranslated strings at transifex website are lost translations that only require to review and use the suggested string, so total left to translate words is wrong. Because of this, I have reduced the budget from ~3300 * 0.07 = 230 USD to 150USD per language.

![imagen](https://user-images.githubusercontent.com/50149324/95432819-71213480-094f-11eb-97ef-f1a8fb8468a5.png)

Although total estimated expense for pending translations is 2450 + 450 = 2900USD, we should not spend more than **1500USD** in one cycle. French, Japanese, Chinese and Spanish translations are actually being worked on, which would imply (if all tasks are complete) spending about 1300USD.
As always, please get in contact with me before start translating. Brazilian Portuguese should be translated as well, as it's on of the most active Bisq markets.

**Comment by stopfeds** [comment]
> As always, please get in contact with me before start translating. Brazilian Portuguese should be translated as well, as it's on of the most active Bisq markets.

@MwithM I sent you a message on Keybase

**Comment by MwithM** [comment]
### Cycle 18 report

German and Chinese desktop projects are being translated, maybe Portugues brazilian as well (I've been contacted for it through Keybase but haven't seen transaltions performed). 
Estimated bisq-website expenses. Many strings were already translated but got missing and now can be found at suggestions.

Spanish bisq-desktop translation had many strings already translated but changed to add hyperlinks, now are marked with label "hyperlinks".

**Comment by MwithM** [comment]
### Budget for Cycle 19

![imagen](https://user-images.githubusercontent.com/50149324/98513193-cb235b80-2267-11eb-89d2-e3dc93f491d3.png)


![imagen](https://user-images.githubusercontent.com/50149324/98512302-661b3600-2266-11eb-8bb1-00b80d1d3e77.png)

Allocated budget for Cycle 19 is 1806+450 = 2256USD. As most of the expense is for languages with no clear dedicated contributors, this amount won't be totally executed, by far.

Japanese bisq-desktop translation has been completed. @ripcurlx could you integrate this, together with Italian and Czech?

**Comment by ripcurlx** [comment]
> Japanese bisq-desktop translation has been completed. @ripcurlx could you integrate this, together with Italian and Czech?

Sure. To keep track could you please create an issue for it and I'll pick it up from there.

**Comment by ripcurlx** [comment]
Just checked the code: Japanese is already integrated in the desktop client. I'll add Czech and Italian though.

**Comment by ripcurlx** [comment]
New languages added already in the release branch at https://github.com/bisq-network/bisq/commit/222ee4029817e7848a6aca2ca7a0182cc9903121

**Comment by MwithM** [comment]
Already translated languages (Spanish, Chinese...) have seen an unexpected grow in strings. Although some are already translated, there's been a rephrase for others and completely new strings.
The budget reflected in my last post should be increased. For example, in Spanish, for the work I've done until this moment i would ask 50USD instead of the expected 25.

**Comment by MwithM** [comment]
### Cycle 19 report

Lots of thanks for implementing Czech, Italian and Japanese, and thanks again to Czech translator petrgremlica for translating a language out of the core ones.
German, Spanish and Chinese have been updated this cycle, together with Japanese which was never completed before.

**Comment by MwithM** [comment]
### Current expenses on Cycle 19

A total of 850USD has been expended on Cycle 19.

![imagen](https://user-images.githubusercontent.com/50149324/101749785-6d846800-3ace-11eb-8c03-57cc63198543.png)

Japanese, German, Spanish, Chinese, French desktop translations are up to date. Czech is as well.

### Budget for Cycle 20

![imagen](https://user-images.githubusercontent.com/50149324/101752678-0f598400-3ad2-11eb-9dbd-fc4dc23451c8.png)
![imagen](https://user-images.githubusercontent.com/50149324/101752583-fd77e100-3ad1-11eb-8601-fa89324acba9.png)

A total amount of 1290 + 450 = **1740USD** is allocated for Cycle 20. This is the max total amount and every task will need to be individually approved.

Consult budget spreadsheet at: https://drive.google.com/file/d/1DdzjrQtD9PAt2wbkUp4z3AwAJhv1JI9S/view?usp=sharing

**Comment by MwithM** [comment]
### Cycle 20 report

Unfortunately, no news from french translator @rafael2333 and very little activity for this cycle. I accepted a Polish translator and Czech is being translated on a daily basis, as it should be for core languages.

**Comment by stevenkain** [comment]
Still no budget for Romanian I see?

**Comment by MwithM** [comment]
### Budget for cycle 21

DAO vote [failed](https://github.com/orgs/bisq-network/teams/dao/discussions/) on cycle 20 and the approved proposals could not receive the corresponding compensation. The amount left to compensate for Cycle 21 was 216 USD.

Budget for Desktop:
![imagen](https://user-images.githubusercontent.com/50149324/104579542-5e0fb600-565c-11eb-818e-76041dc6b802.png)

Budget for Website:
![imagen](https://user-images.githubusercontent.com/50149324/104579896-dfffdf00-565c-11eb-850d-ebb6cfc36b45.png)

Total budget for tasks done at Cycle 21 is 1780 USD.

**Comment by MwithM** [comment]
#### Cycle 20 report

Desktop: Japanese, Spanish and German have been translated. Chinese might be translated as well.

I called translators to translate before next release so new rules regarding Trade ID as reason of payment could be translated, but I don't think I succeeded.

**Comment by MwithM** [comment]
### Cycle 21 report

644 USD were spent between cycles 20 and 21.

### Cycle 22 budget

Desktop

![imagen](https://user-images.githubusercontent.com/50149324/108040158-418ce200-703d-11eb-9a07-6c8c4d1ddad2.png)

Website

![imagen](https://user-images.githubusercontent.com/50149324/108040300-6e40f980-703d-11eb-8f72-60b80deaeca1.png)

Total allocated for Cycle 22 is 1650USD.

**Comment by MwithM** [comment]
## Cycle 22 report.

New translator Deti_P joined to improve German translations and will not ask compensation for that. Thanks.

Very little activity to report on translations for this cycle. I translated the new Spanish strings for desktop.

**Comment by comradekingu** [comment]
If you want more and better translations, move away from Transifex.
I'll help out if you use Weblate, and I'll throw in a nb_NO translation.

**Comment by MwithM** [comment]
- Only 39$ were spent at Cycle 22. Available to spend won't go over 6000USD.
- A single one branch from Pt and Pt_br will be continued as a Bisq core language. To me, it was an error to separate this into two different languages when not even wikipedia does it.
- Yodawarrior @paulmercier-bisq started translating to French.

### Cycle 23 budget


![imagen](https://user-images.githubusercontent.com/50149324/112462787-c0261f00-8d61-11eb-81ae-a2e04e557938.png)
![imagen](https://user-images.githubusercontent.com/50149324/112462670-9c62d900-8d61-11eb-969a-563938e538be.png)

**990USD** is the budget for this cycle.

**Comment by MwithM** [comment]
Yodawarrior (@paulmercier-bisq) has translated all French strings left. Cost is 390USD.
@arnaudfr86 will review all French strings for 120USD, I have agreed to a bounty for it after asking to take a look at Yodawarrior's translation.

**Comment by MwithM** [comment]
### Cycle 23 report

As announced previously, I accepted a bounty for reviewing French. This Cycle only French and Spanish had translating activity.
Many new users wanted asked permission to translate, but didn't reach out at Keybase and I did set this requirement to accept new translators for core languages.

**Comment by MwithM** [comment]
452 USD were spent on Cycle 23, mainly for French.

### Cycle 24 budget
![imagen](https://user-images.githubusercontent.com/50149324/115363145-7f83bf00-a1c2-11eb-83e2-dffb1e9f7df5.png)
![imagen](https://user-images.githubusercontent.com/50149324/115363276-9c1ff700-a1c2-11eb-8c33-c3430fdda534.png)

**Comment by MwithM** [comment]
### Cycle 24 report

Not much activity, AFAIK only Spanish was translated.

**Comment by MwithM** [comment]
### Cycle 25 report

No special activity to report.

**Comment by MwithM** [comment]
### Cycle 26 report

At this cycle, translations have been on hold, I did not even create a budget for it.

**Comment by MwithM** [comment]
### Cycle 27 report

Same as 26, no budget and no translations.

**Comment by MwithM** [comment]
### Cycle 28 report

Nothing to report.

**Comment by MwithM** [comment]
### Cycle 29

Discussion about adding Hindi to Bisq, but has been rejected since we expect that first Bisq users will be fluent in English. Language could be one of the last hurdles for new users.
As last cycle, there has been no activity in this cycle.

**Comment by MwithM** [comment]
### Cycle 30

Czech is the only fully translated source..
One translator contacted at transifex for German.

**Comment by MwithM** [comment]
### cycle 31

- Spanish will be fully translated this cycle and I plan to keep up with core translations, updating the budget and finding translators.

**Comment by MwithM** [comment]
### Cycle 32 report

- Budget has will be updated soon, only Spanish has been up to date. There's several strings referring to payment methods that feels like it's not necessary to translate to most (i.e. Indian payment methods to Spanish or French) marked with the tag `no_translate`.
- While translating, I placed a couple pull requests to improve the original strings: https://github.com/bisq-network/bisq/pull/5965 and https://github.com/bisq-network/bisq/pull/5884

**Comment by MwithM** [comment]
### Cycle 33 report

- Italian translation, among most core translations are on hold at least until Bisq2 is ready.
- 210 USD were spent for Spanish translation on Cycle 32.
